projects/plugins/jetpack/modules/shortcodes/flickr.php

```diff
@@ -217,6 +217,11 @@ function flickr_shortcode_video_markup( $atts, $id, $video_param ) {
 
 		$embed_page = wp_remote_get( $embed_url );
 
+		// Bail if the request returns an error.
+		if ( is_wp_error( $embed_page ) ) {
```

Comment:
Nice! I wonder if we want to explicitly check for array here (minor / not blocking)
```suggestion
		if ( ! is_array( $embed_page ) ) {
```

Reply:
That works for me. 👍